bitnami/elasticsearch/templates/coordinating-statefulset.yaml

```diff
@@ -135,9 +156,9 @@ spec:
             timeoutSeconds: {{ .Values.coordinating.startupProbe.timeoutSeconds }}
             successThreshold: {{ .Values.coordinating.startupProbe.successThreshold }}
             failureThreshold: {{ .Values.coordinating.startupProbe.failureThreshold }}
-            httpGet:
-              path: /_cluster/health?local=true
-              port: 9200
+            exec:
+              command:
+                - /opt/bitnami/scripts/elasticsearch/healthcheck.sh
```

Comment:
Why this change?

Reply:
Because Probes will fail if X-Pack security is enabled. All API endpoints, including `/_cluster/health?local=true` will be password-protected and fail because of authentication failed (code 401).

The new Elasticsearch image includes a `healtcheck.sh` with the logic for the Probes as Kubernetes does not provide mechanisms to perform an `httpsGet` with authentication and using command would require the password to be provided inline (not secret) and using the env variable ELASTICSEARCH_PASSWORD doesn't work either.